tests/acceptance/edit-reminder-test.js

```diff
@@ -72,3 +80,13 @@ test('tay-tay class toggles when reminder has dirty data', function(assert) {
     assert.equal(find('.tay-tay').length, 0, 'should no longer have tay-tay class when dirty data exists')
   })
 })
+
+test('delete button functionality', function(assert) {
+  click('.delete-reminder-btn')
```

Comment:
Will you confirm the reminder is there and what URL you are at. Please test pages that have delete buttons.